<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScene</class>
 <widget class="QWidget" name="GameScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shapez</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <widget class="QLabel" name="coin_count_label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>150</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="help_btn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/help.png</normaloff>:/res/help.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="shop_btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/shop.png</normaloff>:/res/shop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="miner_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/miner_button.png</normaloff>:/res/miner_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="conveyor_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/belt_button.png</normaloff>:/res/belt_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="cutter_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/cutter_button.png</normaloff>:/res/cutter_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="transhcan_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/trash_button.png</normaloff>:/res/trash_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="eraser_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/eraser.png</normaloff>:/res/eraser.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="round_ore_label">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rect_ore_label">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="left_half_label">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="right_half_label">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round_ore_label_t">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>1</y>
     <width>61</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rect_ore_label_t">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>1</y>
     <width>61</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="left_half_label_t">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>1</y>
     <width>61</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="right_half_label_t">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>1</y>
     <width>61</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="rotater_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/rotater_button.png</normaloff>:/res/rotater_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="merger_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/merger_button.png</normaloff>:/res/merger_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
